packages/block-editor/src/components/block-list/breadcrumb.native.js

```diff
@@ -27,8 +28,9 @@ const BlockBreadcrumb = ( {
 } ) => {
 	return (
 		<View style={ styles.breadcrumbContainer }>
-			<TouchableOpacity
-				style={ styles.button }
+			<PrimitiveButton
```

Comment:
These changes are only for demonstrating how to style component using`styled-system` props